app/src/main/res/values/styles.xml

```diff
@@ -3,6 +3,11 @@
 
     <style name="DrawingTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
         <item name="myBackground">@color/arsenic</item>
+        <item name="toolBarBackground">@color/white</item>
+        <item name="iconFillColor">@color/black</item>
+    </style>
+
+    <style name="LayoutBackground">
```

Comment:
There's no need to have "android:background" defined and there's definitely no need to have a whole different style just for a background. Just use your attribute in the layout as the background color

Reply:
Adjusted as pointed out here